container/docker/handler.go

```diff
@@ -412,6 +432,21 @@ func (self *dockerContainerHandler) GetContainerLabels() map[string]string {
 	return self.labels
 }
 
+func (self *dockerContainerHandler) GetContainerIPAddress() string {
+	return self.ipAddress
+}
+
+func (self *dockerContainerHandler) ReadFile(filepath string) ([]byte, error) {
+	pid := self.pid
+	filePath := path.Join(self.rootFs, "/proc", strconv.Itoa(pid), "/root", filepath)
+	glog.V(3).Infof("Trying path %q", filePath)
```

Comment:
nit: V(4) for debug info. Add more context (e.g. `"Attempting to read file at %s"`)